bmq-sdk/src/main/java/com/bloomberg/bmq/QueueOptions.java

```diff
@@ -226,15 +236,13 @@ public String toString() {
                 .append(", suspendsOnBadHostHealth: ")
                 .append(getSuspendsOnBadHostHealth())
                 .append(" ]");
-
         return sb.toString();
     }
 
     /** A helper class to create immutable {@code QueueOptions} with custom settings. */
     public static class Builder {
-        private Optional<Long> maxUnconfirmedMessages;
-        private Optional<Long> maxUnconfirmedBytes;
-        private Optional<Integer> consumerPriority;
+        private final Subscription.Builder defaultSubscriptionBuilder;
+        private final HashMap<Integer, Subscription> subscriptions;
```

Comment:
I suppose it should be `Set` now instead of `Map`?

Reply:
Need **mapping** `sId` -> `Subscription`, but not in `QueueOptions`.